spec/test_app/app/views/feature_spec_views/tooltip.html.haml

```diff
@@ -134,4 +134,11 @@
 %hr
 
 = ui.nfg :typeface, subheading: 'Illustration', class: 'mb-1'
-= ui.nfg :illustration, :sm, image: 'test_app/sample_illustration.png', tooltip: feature_spec_tooltip_text, describe: 'tooltip-illustration'
\ No newline at end of file
+= ui.nfg :illustration, :sm, image: 'test_app/sample_illustration.png', tooltip: feature_spec_tooltip_text, describe: 'tooltip-illustration'
+
+%hr
+
+= ui.nfg :avatar, tooltip: feature_spec_tooltip_text, describe: 'tooltip-avatar', body: 'Avatar'
```

Comment:
@jonathanroehm note to self: this needs to be built out as an official avatar before release.